### PR TITLE
AC-868: Updated Address EditText fields

### PR DESCRIPTION
**Pull request by amtheshubham**
AC-868: Address Edittext hint should mention "Address Line" instead of "Address"

## Description of what I changed

In the Register Patient Activity, in the Edittext field of Address there should be a hint like "Address Line 1", "Address Line 2" instead of only "Address".

Before: Address 1, Address 2
Now: Address Line 1, Address Line 2

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-868

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- No new tests required.

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### Screenshots
**Old**

Old
**New**

New

### CODE REVIEW

**Comment by HerbertYiga** [comment]
thanks @amtheshubham

**Review by HerbertYiga — APPROVED** [review]

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -47,8 +47,8 @@
     <string name="reg_ques_dob">Patient\'s birth date*</string>
     <string name="label_or">OR</string>
     <string name="reg_ques_address">Address</string>
-    <string name="addr1_hint">Address 1*</string>
-    <string name="addr2_hint">Address 2</string>
+    <string name="addr1_hint">Address Line 1*</string>
+    <string name="addr2_hint">Address Line 2</string>
```
Comment (rishabh-997) [inline openmrs-client/src/main/res/values/strings.xml:51]: Update this line in Hindi strings.xml too

**Review by rishabh-997 — CHANGES_REQUESTED** [review]
Also, please don't send -4 PR's for issues like this. Go Through the git-rebase tutorial and see how to correct git history. Multiple Pr's is not a good practice.

**Comment by amtheshubham** [comment]
@rishabh-997 I know how to rebase and all but as explained I ran into some other problem which couldn't be fixed. However, I will take care. I have updated Hindi strings.xml file.

**Review by rishabh-997 — APPROVED** [review]
LGTM

**Pull request merged**